src/documents/Invokers.md

```diff
@@ -15,7 +15,7 @@ around the component tree and within the argument list.
 
 ## Types of Invokers ##
 
-###Standard invoker binding to a function using `funcName`, `func` ###
+###Standard invoker binding to a function using funcName, func ###
```

Comment:
This renders just fine, but since you're cleaning things up, there could be a space before "Standard" here